platform/src/main/java/org/stellar/anchor/platform/service/SimpleInteractiveUrlConstructor.java

```diff
@@ -48,6 +57,8 @@ public String construct(Sep24Transaction txn, Map<String, String> request) {
 
   @SneakyThrows
   String constructToken(Sep24Transaction txn, Map<String, String> request) {
+    forwardKycFields(request);
```

Comment:
It's interesting that constructing a JWT could fail due to a network error. What do you think about moving `forwardKycFields` out of this method and requiring `Sep24Service` to call it explicitly?

Reply:
`forwardKycFields` is a anchor platform specific implementation. And this is related to interactive URL construction. That is why it is placed in the constructor. 

It can fail if the configuration is enabled and the `PUT /customer` endpoint is not enabled.

Reply:
Further thinking about it, `forwardKycFields` can also be moved up to the `construct` method.